static/app/views/explore/multiQueryMode/queryConstructors/groupBy.tsx

```diff
@@ -46,7 +47,13 @@ export function GroupBySection({query, index}: Props) {
   return (
     <Section data-test-id={`section-group-by-${index}`}>
       <SectionHeader>
-        <SectionLabel underlined={false}>{t('Group By')}</SectionLabel>
+        <Tooltip
+          title={t(
+            'Aggregated data by a key attribute to calculate averages, percentiles, count and more.'
```

Comment:
Nit: Some of our tooltips describe _what_ the thing is whereas others describe what the _result_ would be (this one does the latter)

Reply:
Perhaps?
```suggestion
            'Aggregate data by a key attribute to calculate averages, percentiles, count and more.'
```